pyvrp/tests/helpers.py

```diff
@@ -1,12 +1,33 @@
 import pathlib
 import time
 from functools import lru_cache
+from typing import List
 
-from pyvrp import Solution
+from pyvrp import ProblemData, Solution, VehicleType
 from pyvrp.read import read as _read
 from pyvrp.read import read_solution as _read_solution
 
 
+def make_heterogeneous(data: ProblemData, vehicle_types: List[VehicleType]):
```

Comment:
That is really for @leonlan indeed, shall we not keep this PR hinge on that? Still, for unit testing I find it very convenient to just make a heterogeneous version of the existing data. I think having a file is nice if there is much data that is irrelevat to the test, but in this case sometimes test logic depends on the capacities used for different types, I think it's more clear to have that in the test rather than multiple seperate data files. Is there really a problem with using this pattern?